### PR TITLE
Add and remove items from To Do List

### PR DESCRIPTION
**Pull request by GloireIrenge**
This part of the project is made with the following changes and functionalities :

- [ ] I removed all the hardcoded items from the todo array and created a new JavaScript file for the add and remove functionalities.
- [ ] I Implemented a function for adding a new task  and a function for deleting a task.
- [ ] In the index.js file, I Implemented a function for editing todo descriptions.
- [ ] When an item is deleted, the remaining item is updated.
- [ ] All changes are saved to the local storage.

### CODE REVIEW

File: src/Add_remove.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
+
+  addTodo = (task, todos) => {
+    todos.push(task);
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+
```
Comment (codepantha) [inline src/Add_remove.js]: - [ ] At the moment, when a to-do task is saved to localStorage, the `index `property is always set to `0`. This is not the expected behaviour. The `index` property for each to-do item should be set to the value of the new array length. Please implement this requirement 🙏 
Kindly see the bug below 👇 
image

File: src/Add_remove.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  deleteTodo = (e, todos) => {
+    e.target.parentElement.remove();
+    todos.splice(e.target.parentElement.id, 1);
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+}
```
Comment (codepantha) [inline src/Add_remove.js]: - [ ] When a to-do list item is deleted, please remember to update all remaining items' indexes, so they represent the current list order and are unique 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

## Hi @GloireIrenge 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Correct GitHub Flow
✔️ Descriptive PR
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Hi @GloireIrenge, 👋 

## Highlights 👏 

- Nice job doing the suggested changes, great! 😎 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**